### DISCUSSION TITLE
Assign teams to a project - require an own permission role

### DISCUSSION THREAD

**Discussion by Flo-MF** [kimai/kimai#3660] [Questions & Answers]
### Describe the problem

Currently, it's not possible, to give anybody the permission, to assign projects to a specific team.
To do so, you require the role "view_all_data" - with no other role you have the opportunity to add or remove teams from a specific project, activity or customer.
But as described in the documentations, it's not recommended, to assign these role ("view_all_data") to any user instead an admin, because you can see all data. ;)

<img width="340" alt="image" src="https://user-images.githubusercontent.com/66617442/204862700-6b7a1543-79c1-431a-af01-251f7b4106cf.png">

### Describe the solution you'd like

an own role, which control these permission

**Comment by kevinpapst** [comment]
<img width="299" alt="Bildschirmfoto 2022-11-30 um 18 13 30" src="https://user-images.githubusercontent.com/533162/204863759-54eaff07-09ad-41f2-98b9-b1550962065a.png">

**Reply by Flo-MF** [reply]
Mit der Berechtigung sieht es halt so aus:

![image](https://user-images.githubusercontent.com/66617442/205069390-e7d4e233-8fd6-4543-bd08-6eaf3048dcab.png)

**Reply by kevinpapst** [reply]
Your user is likely not teamlead (both role and lead of the teams you want to assign).
